File: app/src/main/java/ru/golden/kotlin_weather_station/dto/Coord.java
package ru.golden.kotlin_weather_station.dto;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

class Coord implements Serializable {

    @SerializedName("lon")
    public Float lon;

    @SerializedName("lat")
    public Float lat;
}

File: app/src/main/java/ru/golden/kotlin_weather_station/dto/Coord.kt
package ru.golden.kotlin_weather_station.dto

import com.google.gson.annotations.SerializedName

import java.io.Serializable

data class Coord(
	@SerializedName("lon")
	val longitude: Float?,

	@SerializedName("lat")
	val latitude: Float?
) : Serializable
